yoyodyne/data/tsv.py

```diff
@@ -118,7 +122,11 @@ def _get_symbols(string: str, sep: str) -> List[str]:
         return list(string) if not sep else string.split(sep)
 
     def source_symbols(self, string: str) -> List[str]:
-        return self._get_symbols(string, self.source_sep)
+        symbols = self._get_symbols(string, self.source_sep)
+        # If not tied, then we distinguish the source vocab with {...}.
+        if not self.tie_embeddings:
```

Comment:
maybe easier to say:

```python
if self.tie_embeddings:
   return symbols
...
```

Less negation -> less cognitive load IMO. Also it's the default and that's another good heuristic for what goes first in a conditional.